### PR TITLE
New FPGA Code Sample "latency_control"

### PR DESCRIPTION
**Pull request by shuoniu-intel**
# Adding a New Sample "latency_control"

## Description

Create new code sample to demonstrate how to set latency constraints to pipes and LSUs accesses.

## Checklist
Administrative
- [x] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): yuguen-intel

Code Development
- [x] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [x] Adhere to readme template 
- [x] Enforce format via clang-format config file
- [x] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [x] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [x] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com

Review
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth(@tomlenth) and/or Project Manager

### CODE REVIEW

**Review by yuguen-intel — APPROVED** [review]
This PR has been reviewed internally and manually tested.
Adding @tomlenth for the README and @pmpeter1 for the SYCL code as this is a new sample.

File: DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+## Running the Sample
+
+ 1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+     ```
+     ./latency_control.fpga_emu     (Linux)
+     latency_control.fpga_emu.exe   (Windows)
+     ```
+2. Run the sample on the FPGA device:
+     ```
+     ./latency_control.fpga         (Linux)
```
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/README.md:257]:
```suggestion
     ./latency_control.fpga         (Linux)
     latency_control.fpga.exe     (Windows)
```
We just added this in https://github.com/oneapi-src/oneAPI-samples/pull/968

**Review by yuguen-intel — DISMISSED** [review]

**Review by tomlenth — COMMENTED** [review]
Minor issue, does it make sense to spell out LSU at first use: Load-Storage Unit. We do this in our documentation I noticed. https://www.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide/top/quick-reference/fpga-lsu-controls.html

**Comment by shuoniu-intel** [comment]
> Minor issue, does it make sense to spell out LSU at first use: Load-Storage Unit. We do this in our documentation I noticed. https://www.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide/top/quick-reference/fpga-lsu-controls.html

That's a good point. I will spell it out.

**Review by tomlenth — APPROVED** [review]

**Review by yuguen-intel — APPROVED** [review]

**Review by pmpeter1 — APPROVED** [review]

**Pull request merged**